en/controllers/components/flash.rst

```diff
@@ -85,10 +85,25 @@ The code above will use the warning.ctp element under **plugins/PluginName/src/T
 for rendering the flash message.
 
 .. note::
+    By default, CakePHP escapes the content in flash messages for security
+    reasons. If you are using any request or user data in your flash messages
+    those are escaped and therefore safe to be printed. If you want to output
+    HTML, you need to pass in an `escape` param and also adjust the templates to
```

Comment:
Do we need double char here for code pieces? I forgot this is not real markdown :P